proposals/raster_to_vector.json

```diff
@@ -0,0 +1,53 @@
+{
+    "id": "raster_to_vector",
+    "summary": "Converts a raster to a vector",
+    "description": "Converts a raster data cube into a vector data cube. The bounding polygon of similar areas of pixels is constructed.",
+    "categories": [
+        "cubes"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "A data cube.",
+            "schema": {
+                "type": "object",
+                "subtype": "datacube"
+            }
+        },
+        {
+            "name": "projection",
+            "description": "Coordinate reference system to reproject the converted vector. Specified as an [EPSG code](http://www.epsg-registry.org/) or [WKT2 CRS string](http://docs.opengeospatial.org/is/18-010r7/18-010r7.html).\nThe default value is of the  input raster cube.",
+            "schema": [
+                {
+                    "title": "EPSG Code",
+                    "type": "integer",
+                    "subtype": "epsg-code",
+                    "minimum": 1000,
+                    "examples": [
+                        3857
+                    ]
+                },
+                {
+                    "title": "WKT2",
+                    "type": "string",
+                    "subtype": "wkt2-definition"
+                }
+            ],
+            "optional": true,
+            "default": null
```

Comment:
default is `null` but, that does not match the schema above, is that valid @m-mohr ?

Reply:
Indeed, null needs to be specified in the schema explicitly.

Reply:
Noted, I will actually remove the default from being null as when this optional field is not specified then the CRS of the output vector data will be the same as the input raster data.

Reply:
Optional fields always need a default value though!

Reply:
Okay, so in this guess default will vary based on the projection of the input raster, not sure how I should define that.

Reply:
The default can be `null`, but you have to add that as one of the options in the schema above

Reply:
Okay noted, will update it to that effect.